### ISSUE TITLE
Release Progress `v1.44.0` 💫

### ISSUE THREAD

**Issue by adzialocha** [deltachat/deltachat-desktop#3723]
- [ ] DeltaChat Website (https://github.com/deltachat/deltachat-pages/pull/805)
- [x] flathub (https://github.com/flathub/chat.delta.desktop/pull/125)
- [x] GitHub release (https://github.com/deltachat/deltachat-desktop/releases/tag/v1.44.0)
     - [ ] downloadable files also on GitHub
- [ ] MacOS store (in review)
- [X] Windows store
- [ ] homebrew
- [ ] aur
- [ ] nix https://github.com/NixOS/nixpkgs/pull/289215
- [x] (community) snap

**Comment by Simon-Laux** [comment]
the versions on the download server are blocked by https://github.com/deltachat/deltachat-desktop/issues/3728, it looks like we need a 1.44.1 for fixing it.

**Comment by Simon-Laux** [comment]
Superseded by #3731
